### PR TITLE
No asserts or from chai

### PR DESCRIPTION
**Pull request by zalun**
solves #97

### CODE REVIEW

File: test/testClient.js [hunk 1]
```diff
@@ -32,11 +35,11 @@ describe('mercurius clients support', function() {
     request(mercurius.app)
     .get('/devices/' + token)
     .send()
+    .expect(200)
     .expect(function(res) {
-      assert.equal(res.status, 200);
-      assert.equal(res.body.clients.length, 1);
-      assert.equal(res.body.clients.indexOf('clientXZ'), 0);
-      assert.equal(res.body.machines.machineXZ.clients.clientXZ, '1');
+      res.body.clients.length.should.equal(1);
```
Comment (marco-c) [inline test/testClient.js:40]: IIRC there's some way to directly check the length (something like array.should.have.length).

File: test/testClient.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 var mercurius = require('../index.js');
 var request = require('supertest');
-var assert = require('assert');
+var chai = require('chai');
 var nock = require('nock');
 var testUtils = require('./testUtils.js');
 
+chai.should();
+var assert = chai.assert;
```
Comment (marco-c) [inline test/testClient.js:8]: Can you remove also the remaining assertion?

Comment (marco-c) [inline test/testClient.js:8]: Also in testUnregisterMachine.

Comment (zalun) [inline test/testClient.js:8]: it ised for isDone() - not sure how to do it ...

**Pull request merged**